tiledb/ml/readers/pytorch.py

```diff
@@ -80,6 +80,9 @@ def __init__(
             y_array.schema.attr(idx).name for idx in range(y_array.schema.nattr)
         )
 
+        if buffer_size is not None and buffer_size < batch_size:
```

Comment:
Simpler: `if self.buffer_size < self.batch_size`

Reply:
mypy does not allow it `Unsupported operand types for > ("int" and "None")`